### ISSUE TITLE
SimpleCircleEye Class not working properly.

### ISSUE THREAD

**Issue by devameerhamza** [Bacon/BaconQrCode#51]
This Path code create a circle with a weird edge can some guide me to fix it?
```
return (new Path())
			->move(1.5, 0)
			->ellipticArc(1.5, 1.5, 0., false, true, 0., 1.5)
			->ellipticArc(1.5, 1.5, 0., false, true, -1.5, 0.)
			->ellipticArc(1.5, 1.5, 0., false, true, 0., -1.5)
			->ellipticArc(1.5, 1.5, 0., false, true, 1.5, 0.)
			->close();
```
http://prntscr.com/jyxvd6

**Comment by DASPRiD** [comment]
Does this happen only with a specific renderer or with all renderers? I just created a circle with the same definitions in Inkscape, and it looks correct there:

```
M 1.5,0
A 1.5,1.5 0 0 1 0, 1.5
  1.5,1.5 0 0 1 -1.5,0
  1.5,1.5 0 0 1 0,-1.5
  1.5,1.5 0 0 1 1.5,0
Z
```

**Comment by IMM0rtalis** [comment]
I can reproduce with svg:

```php
$size       = 200;
$encoding   = 'utf-8';

$url = "sdfsdfsdf";

$renderer = new ImageRenderer(
    new RendererStyle(
        $size,
        4,
        SquareModule::instance(),
        SimpleCircleEye::instance()
    ),
    new SvgImageBackEnd()
);

$bacon = new BaconQrCodeWriter($renderer);
$data = $bacon->writeString($url, $encoding);
$bla = 'data:image/svg+xml;base64,'.base64_encode($data);

echo "<img src='$bla' />";
```

**Comment by DASPRiD** [comment]
Please validate again against `master`, a fix for some matrix transformations was just merged.

**Comment by anhvietcr** [comment]
@devameerhamza Can you draw circle dot path (not eye), like your's picture ? please tell me how to make with this lib.

**Comment by DASPRiD** [comment]
Fixed by #69
